### ISSUE TITLE
Chromium not starting in latest version

### ISSUE THREAD

**Issue by petermanders89** [symfony/panther#262]
I am using panther for a while now, and it is working nicely. However since I updated to 0.60.0 chromium is not launching anymore. I am using a docker image php:7-fpm-buster. 

Previously chromium 76 was installed. When running the tests and inspecting the docker container I could see that indeed chromium was started. Since the latest version, using chromium 78, it does not launch at all. I verified that chromium 78 is installed in the container. The odd thing is that I do not get any error or timeout. I let it run for 15 minutes, and no message. Inspecting the container shows that phpunit is running and php is started. 

Running on the latest version of Symfony (4.3.8) in combination with PHPunit 8.3.5.

**Comment by petermandersvin** [comment]
Slight update, my test did finish in the end, but it took almost 23 minutes. Before it only took about a minute.

**Comment by guillaume-perreal** [comment]
Well, the ChromeDriver has to be version-compatible with Chrome/Chromium (see https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection). This become annoying if you distribute the browser using a pre-built Docker image but ChromeDriver comes from the Composer package. They quickly get out of sync.

I've worked around this problem by providing both the browser and the driver in the image and use the environment variables to have Panther use them both. E.g. :

```Dockerfile
# Install Google Chrome somehow

# Install the version of Chromedriver that is compatible with the installed google-chrome 
RUN CHROME_VERSION=`google-chrome --version | cut -d' ' -f3 | cut -d. -f1-3` \
    && CHROMEDRIVER_VERSION=`curl -sSL https://chromedriver.storage.googleapis.com/LATEST_RELEASE_${CHROME_VERSION}` \
    && curl -sSL https://chromedriver.storage.googleapis.com/${CHROMEDRIVER_VERSION}/chromedriver_linux64.zip -o /tmp/chromedriver.zip \
    && unzip /tmp/chromedriver.zip \
    && mv chromedriver /usr/local/bin/chromedriver \
    && rm /tmp/chromedriver.zip

# Tell Panther to use the downloaded Chromedriver
ENV PANTHER_NO_SANDBOX=1 \
    PANTHER_CHROME_DRIVER_BINARY=/usr/local/bin/chromedriver
```

This should probably work with Chromium.

**Comment by petermandersvin** [comment]
The above helped out, thanks for that. 

However it did not totally solve my issue. I did a fresh Symfony installation in a docker, and there were no issues. So I compared it to my own configuration. It turned out to be the custom light hack provided in https://github.com/symfony/panther/issues/8 to implement code coverage. Removing this decreased my time drastically. I went from 23 minutes to 45 seconds. I did not have this issue with the older version (panther 0.58.0 and chromium 76) though. But this is not officially supported yet so I feel like this is issue can be closed.
